package/MDAnalysis/coordinates/DCD.py

```diff
@@ -172,8 +172,57 @@ def dimensions(self, box):
 
 
 class DCDWriter(base.WriterBase):
-    """Writes to a DCD file
-
+    """Write to a CHARMM/NAMD DCD trajectory file.
+
+    Parameters
```

Comment:
I'd say: open an issue to change it to one or the other and update the style guide according to the consensus.